packages/admin-ui/src/lib/customer/src/components/address-detail-dialog/address-detail-dialog.component.html

```diff
@@ -1,6 +1,5 @@
 <ng-template vdrDialogTitle>
-    <span *ngIf="addressForm.get('streetLine1')?.value as streetLine1">{{ streetLine1 }},</span>
-    <span *ngIf="addressForm.get('countryCode')?.value as countryCode"> {{ countryCode }}</span>
+<span>{{ 'customer.create-new-address' | translate }}</span>
```

Comment:
Hi - this only works for when you are creating a new address, but the same dialog is used when updating an existing address, so in that case the title is wrong.

Reply:
Welcome, then I need to check the edit status and make other title which I think is `customer.edit-address`.